README.md

```diff
@@ -197,6 +197,11 @@ repo:
   storage:
     type: fs
     path: /var/artipie/centos
+  settings:
+    digest: sha256 # Digest algorithm for rpm packages checksum calculation, sha256 (default) and sha1 are supported
+    naming-policy: sha1 # Naming policy for metadata files: plain (default), sha1 or sha256 prefixed
```

Comment:
@olenagerasimova can we use `sha256` as default (same as standard `createrepo` tool for RPM)?